### PR TITLE
Skip a data file

### PR DESCRIPTION
**Pull request by profxj**
By setting skip in the data block, you can skip a specified data file, e.g.

skip LB.20160406.11278.fits

The full path is not required nor any extension.  In princple this
command can cause multiple files to be skipped provided the
input string is in the data file name.

### CODE REVIEW

File: pypit/pypit.py [hunk 1]
```diff
@@ -331,3 +235,153 @@ def load_input(redname):
...
+def load_input(redname, msgs):
+    """
+    Load user defined input reduction file. Updates are
+    made to the argflag dictionary.
+
+    Parameters
+    ----------
+    redname : string
+      Name of reduction script
+
```
Comment (rcooke-ast) [inline pypit/pypit.py]: add docstring for msgs

**Comment by rcooke-ast** [comment]
Looks good to me - feel free to merge when ready!

File: pypit/pypit.py (hunk 1 above)
Comment (profxj) [inline pypit/pypit.py]: added

**Comment by profxj** [comment]
Merging

**Pull request merged**